_posts/2020-11-08-working-with-seqdata-post.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Welcome to Working with Sequence Data!"
+date:   2020-11-08
+categories: 1. ***Overview of FASTA/FastQ***, ***Basic of counting FASTA/FASTQ entries***, ***Trimming low-quality bases***, ***Overview of Parsing FASTA/FASTQ files***
+
+---
+![seq_image](https://i.ytimg.com/vi/fCd6B5HRaZ8/maxresdefault.jpg)
+
+## **FASTA & FASTQ**
+
+### *FASTA*
+- This format is used to store any kind of sequence data which  does not require the per-base pair quality score  
+- FASTA sequence entries have two parts: ***a description and the sequence data, which is denoted by the greater than symbol***. ***The next part contains the sequence which begins on the next line after the description***
+### FASTA Ex
+> ENSMUG00000020122 | ENSMUST00000138518  
```

Comment:
When markdown renders, the special character ">" means you want a block quoted section. It does not literally print the ">" character, which is a hallmark of fasta files. 
As you have it now, this is what the rendered version looks like:
![image](https://user-images.githubusercontent.com/53201954/99887334-4e459980-2bf8-11eb-81a7-a54831327c40.png)
This is not what you want, since it leaves off the important > at the beginning of the sequence ID.
If you want the fasta sequence to still be block quoted, as you have with the fastq example, you need to escape the second > character with a backslash (\) preceding it, like this: 
`>\>ENSMUG00000020122 | ENSMUST00000138518`
Then, when markdown renders, it'll render properly:
![image](https://user-images.githubusercontent.com/53201954/99887388-bc8a5c00-2bf8-11eb-88d1-9765c9aee1b0.png)
Usually there is no space between the > and the start of the sequence ID as well.